docs/release-notes/index.md

```diff
@@ -18,6 +18,43 @@ Review the changes, fixes, and more in each release of Elastic Cloud on Kubernet
 
 ## 3.0.0 [elastic-cloud-kubernetes-300-release-notes]
 
+### Release Highlights
+- ECK 3.0.0 adds support for Elastic stack version 9.0.0. Elastic stack version 9.0.0 is not supported on ECK operators running versions lower than 3.0.0
```

Comment:
I think "stack" has a capital letter in our documentation: https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html
```suggestion
- ECK 3.0.0 adds support for Elastic Stack version 9.0.0. Elastic Stack version 9.0.0 is not supported on ECK operators running versions lower than 3.0.0
```